Taxopark/Taxopark/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Taxopark
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
```

Comment:
JFYI. Regarding the design pattern, the data transfer object doesn't contain business logic. Here you have logic for transformation C temperature to F.